Smartling.API/PublishedFiles/PublishedFilesApiClient.cs

```diff
@@ -40,5 +44,55 @@ public virtual List<PublishedItem> GetRecentlyPublished()
       var result = JsonConvert.DeserializeObject<RecentlyPublished>(response["response"]["data"].ToString());
       return result.items ?? new List<PublishedItem>();
     }
+
+    public virtual RecentlyPublished GetRecentlyPublished(RecentlyPublishedSearch recentlyPublishedSearch)
```

Comment:
It seems the current `GetRecentlyPublished()` can call this method. Just want to minimize code duplicates.

Reply:
@dimitrystd it will be a breaking change so we will need to version it accordingly

Reply:
I meant to leave both methods (with and without params). But actually, no one else uses this API beside sitecore. So I'm ok to leave the only single method.